docs/administration/crud-controller/reference/crud-controller.md

```
@@ -104,11 +105,30 @@ protected function configureQuery(QueryBuilder $queryBuilder): void
+Page titles and the menu label are generated automatically from the entity class name using English singular/plural inflection. For example, entity `OrderItem` produces menu title "Order items", etc.
+
+These generated names are registered as translation keys automatically — no manual `t()` wrapping is needed for defaults.
+
 ### Methods
 
 #### `setTitle(ActionType $actionType, string $title)`
 
-This method allows you to set the title for the given page type. The title is displayed in the page header.
+Sets a custom title for a specific page type. The title is displayed in the page header. Overrides the auto-generated default for the given action.
+
+```php
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)